setup/install/index.md

```diff
@@ -23,10 +23,11 @@ in [Quickstart](/setup/quickstart/) might be a better choice.
 
 Installing a complete Spinnaker involves these steps:
 1. [Install Halyard](/setup/install/halyard/)
-2. [Choose a cloud provider](/setup/install/providers/)
-3. [Choose an environment](/setup/install/environment/)
-4. [Deploy Spinnaker](/setup/install/deploy/)
-5. [Configure everything else](/setup/other_config/) (which includes a lot of
+1. [Choose a cloud provider](/setup/install/providers/)
+1. [Choose an environment](/setup/install/environment/)
+1. [Deploy Spinnaker](/setup/install/deploy/)
+1. [Configure everything else](/setup/other_config/) (which includes a lot of
   stuff you need before you can use Spinnaker in production)
-6. [Productionize Spinnaker](/setup/productionize/) (which mainly helps you
+1. [Productionize Spinnaker](/setup/productionize/) (which mainly helps you
```

Comment:
This isn't technically a word. Perhaps "Use Spinnaker in Production" or "Release Spinnaker to Production."